src/ducks/Studio/timeseries/hooks.js

```diff
@@ -56,6 +57,30 @@ function abortAllMetrics (abortables) {
   return [...abortables.values()].forEach(cancelQuery)
 }
 
+export const getPreparedMetricSettings = (metrics, settings) => {
+  const hasDaaMetric = metrics.includes(Metric.daily_active_addresses)
+
+  if (hasDaaMetric) {
+    metrics.forEach(metric => {
+      settings.set(metric, {
+        interval: '1d'
+      })
+    })
+  } else {
+    settings.forEach((value, key) => {
+      const newValue = {
+        ...value
+      }
+
+      delete newValue['delete']
```

Comment:
@GarageInc what is the reason for deleting the `delete` property?

Reply:
My mistake